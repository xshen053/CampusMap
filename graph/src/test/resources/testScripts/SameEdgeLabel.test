# Create a graph
CreateGraph SEL

# Add a node
AddNode SEL n1
AddNode SEL n2

# Add an edge
AddEdge SEL n1 n1 e
AddEdge SEL n1 n2 e
AddEdge SEL n2 n2 e

# Print the nodes in the graph and the outgoing edges from n1
ListNodes SEL
ListChildren SEL n1
ListChildren SEL n2