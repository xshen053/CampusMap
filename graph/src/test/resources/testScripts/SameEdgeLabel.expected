# Create a graph
created graph SEL

# Add a node
added node n1 to SEL
added node n2 to SEL

# Add an edge
added edge e from n1 to n1 in SEL
added edge e from n1 to n2 in SEL
added edge e from n2 to n2 in SEL

# Print the nodes in the graph and the outgoing edges from n1
SEL contains: n1 n2
the children of n1 in SEL are: n1(e) n2(e)
the children of n2 in SEL are: n2(e)